.env.example

```
@@ -1,11 +1,28 @@
+
+# eureka
+EUREKA_SERVER_PORT=8761
+
+# gateway
+GATEWAY_SERVER_PORT=8080
+
+# hub
+HUB_SERVER_PORT=8084
+
+# hubpath
+HUB_SERVER_PORT=8086
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The HUB_SERVER_PORT key is duplicated

(DuplicatedKey)